keycloak/keycloak-operator
# feat: Adding status updates to KeycloakRealm

## JIRA ID
https://issues.jboss.org/browse/INTLY-3363

## Additional Information
Adding Status fields to KeycloakRealm
- Phase
- Message
- Ready
- SecondaryResources
- LoginUrl

Updated on success and failure

## Verification Steps
1. Start the operator from this branch
2. Run `kubectl create -f deploy/examples/keycloak/keylcoak.yaml`
3. Run `kubectl create -f deploy/examples/realm/realm_with_users.yaml`
4. Run `oc get keycloakrealms example-keycloakrealm -o yaml`
5. Verify the status is the following
```
status:
  loginUrl: ""
  message: ""
  phase: reconciling
  ready: true
  secondaryResources:
    Secret:
    - credential-user-realm-realm-admin-keycloak
``` 

## Checklist:
